tez-api/src/main/java/org/apache/tez/dag/api/TezConfiguration.java

```diff
@@ -2296,4 +2296,12 @@ static Set<String> getPropertySet() {
   @ConfigurationProperty
   public static final String TEZ_MRREADER_CONFIG_UPDATE_PROPERTIES = "tez.mrreader.config.update.properties";
 
+  /**
+   *  Frequency at which thread dump should be captured.
+   */
+  @ConfigurationScope(Scope.DAG)
+  @ConfigurationProperty
+  public static final String TEZ_JOB_PERIODIC_THREAD_DUMP_FREQUENCY = "tez.job.periodic.thread.dump.frequency";
```

Comment:
can we add the unit to the config name for clarity? also, "job" doesn't hold any useful information (we simply make thread dumps in every containers), this can be like:
```
tez.periodic.thread.dump.frequency.ms
```

Reply:
I changed the config name, but didn't suffix ms, as this config can take any time unit, 1s, 10ms anything, if you don't specify unit, it is by default ms.

I am using conf.getTimeDuration for retrieving the value